database/postgresdbConnection.js

```diff
@@ -4,26 +4,27 @@ const sql = postgres('postgres://username:password@host:port/database', {
   host: 'localhost',
   port: '5432',
   database: 'chewy_images',
+  username: 'postgres',
   // update this to use an environment variable if using in production
   password: 'password',
```

Comment:
Even though this is just a local db for now, I would use an environment var here to get in the habit of never putting passwords in github.